styles.css

```
@@ -0,0 +1,311 @@
+        width: 100%;
+        height: auto;
+    }
+    @media screen and (min-width: 550px) {
+    
+        
+        .phone{
+            display:flex ;
+            height: 570px;
+            transform: translateX(32px);
+            margin: -52px;
+        }
+        .container-twocolumns {
+    display: flex;
+    width: 100%;
+    flex-direction: row;
+    justify-content: center;
+    align-items: flex-start;
+    box-sizing: border-box;
+        }
+    }
+ 
+    @media screen and (min-width:768px) {
+        
+        .canvas img {
+        height: 450px; 
+        }
+
+        .menu{
+            display: flex;
+            list-style: none;
+            justify-content: center;
+            align-items: center;
+        }
+        .menu a:hover{
+            text-decoration: underline;
+        }
+        .menu a {
+            color: white;
+            margin: 16px;
+            font-weight: bold;
+            text-decoration: none;
+        }
+.Hamburger{
+    display: none;
+}
+.boton-principal{
+    display: flex;
+    align-items: center;
+    font-size: 17px;
+    letter-spacing: 3px;
+    font-weight: bold;
+    border-radius: 17px;
+    border: none;
+    background-color: rgba(255, 255, 255, 0.343);
+    color: white;
+    padding: 7px;
+    margin-right: 0px;
+    margin-left: -10px;
+}
+    
+    .travesia{
+     
+        display: flex;
+        justify-content: space-between;
+        /* align-items: center; */
+        width:100%;
+        box-sizing: border-box;
+        padding: 15px 20px;
+        margin-bottom: 88px;
+      
+    }
+    .menu{
+        display: flex;
+    }
+    h1{
+        font-family: 'Merriweather Sans', sans-serif;
+        font-size: 65px;
+        margin: 11px;
+        width: 100%;
+        letter-spacing: 1px;
+        line-height: 85px;
+        margin-bottom: 28px;
+        padding-right: 0px;
+    }
+    p{
+        font-size: 20px;
+        line-height: 29px;
+        letter-spacing: 0px;
+        padding-right: 40px;
+        margin-bottom: 25px;
+        margin-inline: 10px;
+        margin-top: -9px;
+    }
+    .patrocinadores-grid{
+        display: grid;
+        grid-template-columns: 1fr 1fr 1fr 1fr;
+        grid-template-rows: 1fr 1fr;
+        margin: 100px -80px 64px 0px;
+        column-gap: 17px;
+        row-gap: 30px;
+        justify-items: start;
+    }
+    
+}
+@media screen and (min-width:1024px) {
+    
+h1{
+    
+    
+    font-size: 70px;
+    padding-right: 160px;
+    padding-left: -10px;
+}
+.phone{
+    margin: -20px;
+    padding-top: 1px;
+}
+  .patrocinadores-grid{
+    margin: 70px -60px 64px 0px;
+    margin-top: 15px;
+
+  }
+  .first-button:hover{
+    background-color: rgba(0, 255, 255, 0.486);
+    color: chocolate;
+  }
+  @keyframes move {
+    from{
+        transform: translateX(-5px);
+    }
+    to {
+        transform: translateX(10px);
+    }
+  } 
+  .boton-principal:hover {
+    background-color:lightcoral;
+  } 
+   .flecha:hover {
```

Dile a tu CSS que cuando le hagas **hover** al botón, le aplique el efecto de transición a la flecha... en este momento toca es pararse exclusivamente sobre la flecha y no tiene sentido si hace parte de un botón. Cópialo tal cual te lo sugiero aquí y me dices si te gusta más ese efecto.

```suggestion
   .first-button:hover .flecha {
```